### DISCUSSION TITLE
Custom accent color

### DISCUSSION THREAD

**Discussion by HalanoSiblee** [WerWolv/ImHex#1748] [Feedback]
Add in the setting interface accent color options to change from default blue to any other color
unlike color theme that change the entire interface scheme.

**Comment by WerWolv** [comment]
Hey! I just added this feature with the latest commit :) Please try it out and let me know what you think! You can change the color under Settings -> Interface -> Accent Color.

<img width="1392" alt="image" src="https://github.com/user-attachments/assets/31d50c04-b683-416e-a337-297716145b43" />
<img width="1392" alt="image" src="https://github.com/user-attachments/assets/b29fe313-b881-4632-9a4a-6a44790f4369" />

**Reply by HalanoSiblee** [reply]
Thank you.
